index.html

```diff
@@ -5,12 +5,175 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Conference</title>
-    <link rel="stylesheet" href="./styles/style.css" />
+    <title>Home</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+    <link rel="stylesheet" href="./styles/common.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+        integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
 </head>
 
 <body>
+    <!-- overlay mobile menu code -->
+    <div id="navbar-overlay" class="overlay-background">
+        <a class="closeIcon"><img src="./assets/close-icon.png" alt="Close icon"></a>
+        <ul class="overlay-content">
+            <li><a class="one" href="about.html">About</a></li>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="#program">Program</a></li>
+            <li><a href="#sponsor">Sponsors</a></li>
+            <li class="menu-item-border"><a class="item-link" href="#">SL campaign </a></li>
+        </ul>
+    </div>
 
+    <!-- DESKTOP: social media header   -->
+    <div class="social">
+        <div> <i class="fab fa-twitter"></i></div>
+        <div> <i class="fab fa-facebook-f"></i></div>
+        <div>English</div>
+        <div>My Page</div>
+        <div>Logout</div>
+    </div>
+
+    <!-- nav -->
+    <header class="nav">
+        <!-- <i class="fas fa-bars" id="humberger"></i> -->
+        <nav class="navbar-humberger">
+            <div class="icon-tab" id="humberger">
+                <div class="tab"></div>
+                <div class="tab"></div>
+                <div class="tab"></div>
+            </div>
+        </nav>
+        <ul class="nav__list">
+            <li class="nav__list__item nav__list__logo"><a href="about.html"><img src="./assets/nav-startup.jpg"
```

Comment:
- [ ] [OPTIONAL]: The standard usually has the logo redirecting the user to the home page, you should consider changing that!
- [ ] If you decide that is still better to have the logo redirecting to the about page, you should add a link to the home page for the desktop version (right now, once you are on the about page, you can't go back by clicking on any link, just by changing the URL);